app/views/play/CampaignView.coffee

```diff
@@ -1558,9 +1558,12 @@ module.exports = class CampaignView extends RootView
     if what in ['level', 'xp']
       return me.showGemsAndXpInClassroom() or !isStudentOrTeacher
 
-    if what in ['settings', 'leaderboard', 'back-to-campaigns', 'poll', 'items', 'heros', 'achievements', 'clans']
+    if what in ['settings', 'leaderboard', 'back-to-campaigns', 'poll', 'items', 'heros', 'achievements']
       return !isStudentOrTeacher and not @editorMode
 
+    if what in ['clans']
+      return !isStudentOrTeacher and not @editorMode and !me.get('isCreatedViaLibrary')
```

Comment:
```suggestion
      return !isStudentOrTeacher and not @editorMode and not me.get('isCreatedViaLibrary')
```

Reply:
Prefer `not` over `!` in CoffeeScript, especially here where `not` is used on same line.